### PR TITLE
appid messages 301-325

### PR DESCRIPTION
**Pull request by 1000TurquoisePogs**

### CODE REVIEW

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 1]
```diff
@@ -1848,7 +1856,9 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0068W":"Failed to set proxy authorizations. Error=
+### ZWED0068W
+
+  Failed to set proxy authorizations. Error=
```
Comment (sakshibobade21) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:1862]: Do we need Error=**%s**

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 2]
```diff
@@ -1896,7 +1912,9 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0072W":"Could not stop manager, error=
+### ZWED0072W
+
+  Could not stop manager, error=
```
Comment (sakshibobade21) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:1918]: Do we need error=**%s** ?

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 3]
```diff
@@ -2388,283 +2486,332 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
   **Reason:**
 
-  TODO
+  When the app-server was beinging to the address shown, it recieved the error EADDRNOTAVAIL or ENOTFOUND. In either case, the app-server was not able to bind to the address and so it will not run until the problem is solved.
 
   **Action:**
 
-  TODO
+  Review the address and check if there if it is valid or if there is some lack of permissions that might explain why these errors were received by the server.
```
Comment (sakshibobade21) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:2654]: Review the address and **check if it is valid** or ...

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 4]
```diff
@@ -3192,7 +3425,9 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0147E":"SAF keyring data was not found for "%s"
+### ZWED0148E
+
+  Exception thrown when reading SAF keyring, e=
```
Comment (sakshibobade21) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:3430]: Do we need to specify the exception here? like **e= %s**

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 5]
```diff
@@ -2388,283 +2486,332 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
-  TODO
+  Review the error to resolve it, and contact support if needed.
 
 
 
-### ZWED0001E":"Error: %s
+### ZWED0002E
+
+  Could not stop language manager for types=_languageName_
```
Comment (DivergentEuropeans) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:2603]: _languageName_ in this case looks like an array, so may be better to rename it to _languageNames_

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 6]
```diff
@@ -2388,283 +2486,332 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
+### ZWED0005E
 
-### ZWED0004E":"Could not listen on address %s:%s. It is already in use by another process.
+  Could not listen on address _ip_:_port_. Invalid IP for this system.
 
   **Reason:**
 
-  TODO
+  When the app-server was beinging to the address shown, it recieved the error EADDRNOTAVAIL or ENOTFOUND. In either case, the app-server was not able to bind to the address and so it will not run until the problem is solved.
```
Comment (DivergentEuropeans) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:2650]: beinging -> binding 
I believe

**Review by DivergentEuropeans — CHANGES_REQUESTED** [review]
These are great, detailed messages, but just a few comments

Also, you probably shouldn't have changed the formatting of the messages above you because these will be overwritten. I hope there aren't annoying merge conflicts

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 7]
```diff
@@ -2388,283 +2486,332 @@ The following error message codes may appear on the app-server log. Use the foll
...
 
   **Reason:**
 
-  TODO
+  The loopback configuration that the server uses to contact itself over an internal network was missing a value for the network port, therefore no requests over the loopback address will be possible.
 
   **Action:**
 
-  TODO
+  Review the configuration of components.app-server.node.port to see if it has a value and set one to fix the issue.
```
Comment (DivergentEuropeans) [inline docs/troubleshoot/app-framework/appserver-error-codes.md:2626]: Btw, it's probably good to code format the components.app-server.node.port

I think it's the ` character in markdown but I could be wrong

File: docs/troubleshoot/app-framework/appserver-error-codes.md (hunk 1 above)
Comment (1000TurquoisePogs) [reply docs/troubleshoot/app-framework/appserver-error-codes.md:1862]: i havent touched these yet, i was just bulk fixing formatting.

File: docs/troubleshoot/app-framework/appserver-error-codes.md (hunk 2 above)
Comment (1000TurquoisePogs) [reply docs/troubleshoot/app-framework/appserver-error-codes.md:1918]: i havent touched these yet, i was just bulk fixing formatting.

File: docs/troubleshoot/app-framework/appserver-error-codes.md [hunk 8]
```diff
@@ -1908,7 +1926,9 @@ The following error message codes may appear on the app-server log. Use the foll
 
 
 
-### ZWED0073W":"No server returned for group=
+### ZWED0073W
+
+  No server returned for group=
```
[the remark this answers is not in the document]
Comment (1000TurquoisePogs) [reply docs/troubleshoot/app-framework/appserver-error-codes.md:1931]: i havent touched these yet, i was just bulk fixing formatting.

File: docs/troubleshoot/app-framework/appserver-error-codes.md (hunk 5 above)
Comment (1000TurquoisePogs) [reply docs/troubleshoot/app-framework/appserver-error-codes.md:2603]: thanks

File: docs/troubleshoot/app-framework/appserver-error-codes.md (hunk 7 above)
Comment (1000TurquoisePogs) [reply docs/troubleshoot/app-framework/appserver-error-codes.md:2626]: it is

File: docs/troubleshoot/app-framework/appserver-error-codes.md (hunk 3 above)
Comment (1000TurquoisePogs) [reply docs/troubleshoot/app-framework/appserver-error-codes.md:2654]: thank you

**Review by DivergentEuropeans — APPROVED** [review]
Looks good 👍

**Pull request merged**